modules/users/manifests/pentestera.pp

```diff
@@ -1,8 +0,0 @@
-# Creates the pentester user
-class users::pentestera {
-  govuk_user { 'pentestera':
-    fullname => 'Pentester A',
```

Comment:
If you want Puppet to actually remove the user more thoroughly, you'll need to make the user `absent` first, before removing the manifest.

See https://docs.publishing.service.gov.uk/manual/removing-a-user-from-puppet.html

Reply:
Ah, yes. I forgot how Puppet works...